app/controllers/candidate_interface/equality_and_diversity_controller.rb

```diff
@@ -10,15 +10,8 @@ def choice
       @choice_form = EqualityAndDiversity::ChoiceForm.new(choice: choice_param)
       render :start and return unless @choice_form.valid?
 
-      entrypoint_path =
-        if current_application.equality_and_diversity_answers_provided?
-          candidate_interface_review_equality_and_diversity_path
-        else
-          candidate_interface_edit_equality_and_diversity_sex_path
-        end
-
       if @choice_form.choice == 'yes'
-        redirect_to entrypoint_path
+        redirect_to candidate_interface_edit_equality_and_diversity_sex_path
```

Comment:
I don't think we need any of this choice form stuff, i've removed in my pr, can rebase after i've merged